### PR TITLE
Add 60 seconds sleep to clear host connections

### PR DESCRIPTION
**Pull request by nbrady-techempower**
tl;dr We believe that some frameworks are hitting the OS limit for open connections and, if the next test spins up too fast, it may have trouble creating network connections. Because we're using `--host`, these connections are lingering on the host and stuck in a `TIME_WAIT` state that automatically close after a default of 60 seconds. Here, we're giving enough time for the host to close these connections before the next test starts. 

🤞 that this solves #4092 

Edit: Changed my mind about where to put this. Rather than before a test container starts, I put it in the `__exit_test` cleanup. This makes sure building database containers and all the other things that need to happen at the start of a test can reach the interwebs.

### CODE REVIEW

File: toolset/benchmark/benchmarker.py [hunk 1]
```diff
@@ -92,6 +92,9 @@ def __exit_test(self, success, prefix, file, message=None):
                 file=file,
                 color=Fore.RED if success else '')
         self.time_logger.log_test_end(log_prefix=prefix, file=file)
+        # Sleep for 60 seconds to ensure all host connects are closed
+        log("Clean up: Sleep 60 seconds...", prefix=prefix, file=file)
+        time.sleep(60)
```
Comment (msmith-techempower) [inline toolset/benchmark/benchmarker.py:97]: Even if it's the only one to run?

Comment (nbrady-techempower) [reply toolset/benchmark/benchmarker.py:97]: Feel free to crtl-C if it's the only one run. Really, I just care about getting this on citrine and seeing if it solves the issues. Then I'll worry about QoL stuff.

Comment (msmith-techempower) [reply toolset/benchmark/benchmarker.py:97]: Fair enough.
